libs/importer/src/importers/bitwarden/bitwarden-password-protected-importer.spec.ts

```diff
@@ -35,6 +40,36 @@ describe("BitwardenPasswordProtectedImporter", () => {
     pinService = mock<PinServiceAbstraction>();
     accountService = mock<AccountService>();
 
+    accountService.activeAccount$ = of({
+      id: newGuid() as UserId,
+      email: "test@example.com",
+      emailVerified: true,
+      name: "Test User",
+    });
+
+    const mockOrgId = "00000000-0000-0000-0000-000000000000" as OrganizationId;
```

Comment:
🎨  Non-blocking: Could use `emptyGuid` from `"@bitwarden/common/types/guid"` as a const instead re-defining this literal.
```suggestion
    const mockOrgId = emptyGuid as OrganizationId;
```